app/controllers/journeys_controller.rb

```diff
@@ -1,10 +1,11 @@
 class JourneysController < ApplicationController
   def show
-    @journey = Journey.published.find_by!(slug: params[:id])
+    @journey = Journey.find_by!(slug: params[:id])
```

Comment:
Toto tu bolo inak naschval, ze chceme len published, treba to rozsirit aj na blank, ale unpublished tam davat nechceme.